devito/core/gpu_openmp.py

```diff
@@ -402,4 +495,8 @@ def _specialize_iet(cls, graph, **kwargs):
         data_manager.place_definitions(graph)
         data_manager.place_casts(graph)
 
-        return graph
+        # Initialize OpenMP environment
+        initialize(graph)
+        mpiOmpizer(graph)
```

Comment:
this `mpi_ompizer` should be dropped and added to `_make_passes_mapper` . This is a Custom Operator. It only applies the passes that users ask it to apply, aside from the vey basic things